### DISCUSSION TITLE
[Feature] Display Video Metadata on Sidebar

### DISCUSSION THREAD

**Discussion by jtamm-red** [immich-app/immich#11278] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The mobile app currently displays more detailed video information compared to Immich. To ensure consistency and improve user experience.

![image](https://github.com/user-attachments/assets/6f5f9325-b7ec-4ceb-873a-5d252542dc2b)

I would like to have video metadata displayed on the sidebar. The metadata should include:
* Video length
* Codec information
* Frame rate

Additionally, I would like to see the mobile model information displayed. It should also be possible to apply filters to this data.

![image](https://github.com/user-attachments/assets/4aa68273-5fe1-470a-90f3-d4fe2d8bc02a)

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by l4kr** [comment]
<img width="1179" height="2556" alt="0C4E15FA-839A-4469-9D88-5B08126C13B0" src="https://github.com/user-attachments/assets/bd8a1220-1a73-418b-96e1-6afba92a135f" />

iOS aswell. There's just... not much information
